cli/src/alluxio.org/cli/cmd/exec/test_hdfs_mount.go

```diff
@@ -0,0 +1,78 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package exec
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"alluxio.org/cli/env"
+)
+
+var TestHdfsMount = &TestHdfsMountCommand{
+	BaseJavaCommand: &env.BaseJavaCommand{
+		CommandName:   "testHdfsMount",
+		JavaClassName: "alluxio.cli.ValidateHdfsMount",
+		ShellJavaOpts: fmt.Sprintf(env.JavaOptFormat, env.ConfAlluxioLoggerType, "Console"),
+	},
+}
+
+type TestHdfsMountCommand struct {
+	*env.BaseJavaCommand
+	path     string
+	readonly bool
+	shared   bool
+	option   string
+}
+
+func (c *TestHdfsMountCommand) Base() *env.BaseJavaCommand {
+	return c.BaseJavaCommand
+}
+
+func (c *TestHdfsMountCommand) ToCommand() *cobra.Command {
+	cmd := c.Base().InitRunJavaClassCmd(&cobra.Command{
+		Use:   "testHdfsMount [--readonly] [--shared] [--option <key=val>] [--path <hdfsURI>]",
```

Comment:
```suggestion
		Use:   "testHdfsMount",
		Args:  cobra.NoArgs,
```
this will ensure user does not accidentally provide any args. the description for flags will be defined as part of registering the flag in the code lines below, so no need to repeat them here.